modules/installation/pages/setup-wizard.adoc

```diff
@@ -1,29 +1,36 @@
-[[ref.webui.admin.wizard]]
+[[Setrup Wizard]]
 = Setup Wizard
 
-Setting up {productname} typically requires some extra steps after installation for common configuration tasks.
+When you have completed your {productname} installation, you can use the setup wizard to complete the last few steps.
+The setup wizard allows you to configure the HTTP proxy, organization credentials, and {suse} products.
 
-The menu:Admin[Setup Wizard] link is displayed when the {productname} {webui} is used for the fist time and can be accessed later at any time by clicking menu:Admin[Setup Wizard].
-On the three tabs configure the HTTP proxy server, organization credentials, and {suse} products.
+The setup wizard is displayed by default when you log in the {productname} {webui} for the first time.
+You can access the setup wizard directly by navigating to menu:Admin[Setup Wizard].
 
-menu:HTTP Proxy:[]::
-If needed configure a proxy server that {productname} will use to access SCC ({suse} Customer Center) and other remote servers here.
-Use `hostname:port` syntax in the menu:HTTP Proxy[HTTP Proxy Hostname:] field if the proxy port is not 8080.
-Clearing the fields disables proxy.
-+
 
-image::admin_proxy_settings.png[scaledwidth=80%]
-+
 
-menu:Organization Credentials:[]::
-Select menu:Admin[Setup Wizard > Organization Credentials > Add a new credential] then enter user name and password to give another organization/user access to {scc}.
-+
+== Configure the HTTP Proxy
 
-image::admin_organization_credentials.png[scaledwidth=80%]
-+
+{productname} can connect to the {scc} or other remote servers using a proxy.
```

Comment:
```suggestion
{productname} can connect to the {scc} (SCC) or other remote servers using a proxy.
```
Later on, we use just "SCC".  I think it's good practice to define the abbrev this way where it occurs for the first time in a chapter or section.